### PR TITLE
XDR file retrieve logic fix

### PR DESCRIPTION
**Pull request by yucohen**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://jira-hq.paloaltonetworks.local/browse/CIAC-2919
Fixing the bug in the readme: https://jira-hq.paloaltonetworks.local/browse/XSUP-13223

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by dansterenson — CHANGES_REQUESTED** [review]

File: Packs/CortexXDR/Integrations/CortexXDRIR/README.md [hunk 1]
```diff
@@ -17,7 +17,6 @@ This integration was integrated and tested with version 2.6.5 of Cortex XDR - IR
     | API Key |  | True |
     | Only fetch starred incidents |  | False |
     | Starred incidents fetch window (&lt;number&gt; &lt;time unit&gt;, e.g., 12 hours, 7 days)  |  | False |
-    | First fetch timestamp (&lt;number&gt; &lt;time unit&gt;, e.g., 12 hours, 7 days)__
```
[the remark this answers is not in the document]
Comment (bziser) [reply Packs/CortexXDR/Integrations/CortexXDRIR/README.md:20]: Fix to this bug: https://jira-hq.paloaltonetworks.local/browse/XSUP-13223

File: Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.py [hunk 2]
```diff
@@ -1188,22 +1195,29 @@ def main():  # pragma: no cover
...
+                                          results_function=action_status_get_command,
+                                          polling_field="status",
+                                          polling_value=["PENDING",
+                                                         "IN_PROGRESS",
+                                                         "PENDING_ABORT"])
+            raw = polling.raw_response
+            status = raw.get('status') if isinstance(raw, dict) else raw[0].get('status')
+            if polling.scheduled_command:
+                return_results(polling)
+            elif status == 'COMPLETED_SUCCESSFULLY':
```
[the remark this answers is not in the document]
Comment (bziser) [reply Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.py:1212]: It is coming from `action_status_get_command` which is used in the polling. 
The value `COMPLETED_SUCCESSFULLY` is used for knowing whether we want to continue to the `'xdr-retrieve-file-details'` command.

File: Packs/CortexXDR/ReleaseNotes/4_6_4.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Palo Alto Networks Cortex XDR - Investigation and Response
```
Comment (bziser) [inline Packs/CortexXDR/ReleaseNotes/4_6_4.md:3]: Please add that you changed also the integration README

**Review by bziser — APPROVED** [review]
From my side, it seems to be good.
Great work :)
Waiting for @dansterenson's review.

**Review by dansterenson — APPROVED** [review]
Good job

**Pull request merged**